lib/cipher_working_notes.rb

```diff
@@ -0,0 +1,23 @@
+#scratch paper for working out Cipher class/method
+#didn't get very far (sad face)
+
+class Cipher
+  attr_accessor :characters, :string, :alphabet
+  def initialize
```

Comment:
Down in your tests you do `cipher = CaesarCipher.new("the quick brown fox jumps over the lazy dog", 3)`. This means your Cipher initialize method should take 2 parameters. Remember, new is an alias for initialize